input/pagecontent/2-9-GetCurrentContext.md

```diff
@@ -12,7 +12,9 @@ GET `base-hub-URL/{topic}`
 
 ### Get Current Context Response
 
-This method returns an object containing the current context of a topic; where the current context is the most recent *-open event for which no *-close event has occurred, according to the Subscriber's subscription.  The current context is made up of one or more "top-level" contextual resources and the type of the anchor context SHALL be in the `context.type` field.  For example, if the current context was established using a [`Patient-open`](3-3-1-patient-open.html) request the returned object will contain `context.type: "Patient"`.  If the current context was created by a [`DiagnosticReport-open`](3-6-1-diagnosticreport-open.html) request the returned object will contain `context.type: "DiagnosticReport"`.  If there is no context currently established, the `context.type` SHALL contain an empty string and the `context` SHALL be an empty array.
+This method returns an object containing the current context of a topic; where the current context is the most recent *-open event according to the Subscriber's subscription.  The current context is made up of one or more "top-level" contextual resources and the type of the anchor context SHALL be in the `context.type` field.  For example, if the current context was established using a [`Patient-open`](3-3-1-patient-open.html) request the returned object will contain `context.type: "Patient"`.  If the current context was created by a [`DiagnosticReport-open`](3-6-1-diagnosticreport-open.html) request the returned object will contain `context.type: "DiagnosticReport"`.  If there is no context currently established, the `context.type` SHALL contain an empty string and the `context` SHALL be an empty array.
+
+At any point in time the Hub may contain multiple contexts in a given topic.  Each context is established with a *-open request and remains in the Hub's record of contexts until a *-close request is accepted for the context.  If one of the established contexts is closed without another being opened, the Hub SHALL return an empty context (which as specified above is indicated by returning an empty string for the `context.type` and an empty array in `context`).  See [Multi-tab Considerations](4-4-multitab-considerations.html) for additional discussion on multiple context scenarios.

```

Comment:
```suggestion
If an established contexts is closed without another being opened, the Hub SHALL return an empty context (which as specified above is indicated by returning an empty string for the `context.type` and an empty array in `context`).  See [Multi-tab Considerations](4-4-multitab-considerations.html) for additional discussion on multiple context scenarios.

```

Explaining  "multiple contexts in a given topic" seems problematic because we defined, in the paragraph above, "current context is the most recent *-open event according to the Subscriber's subscription"